lib/rules/object-curly-newline.js

```diff
@@ -73,19 +73,32 @@ function normalizeOptionValue(value) {
  * Normalizes a given option value.
  *
  * @param {string|Object|undefined} options - An option value to parse.
- * @returns {{ObjectExpression: {multiline: boolean, minProperties: number}, ObjectPattern: {multiline: boolean, minProperties: number}}} Normalized option object.
+ * @returns {{
+ *   ObjectExpression: {multiline: boolean, minProperties: number, consistent: boolean},
+ *   ObjectPattern: {multiline: boolean, minProperties: number, consistent: boolean},
+ *   ImportDeclaration: {multiline: boolean, minProperties: number, consistent: boolean},
+ *   ExportNamedDeclaration : {multiline: boolean, minProperties: number, consistent: boolean}
+ * }} Normalized option object.
  */
 function normalizeOptions(options) {
-    if (options && (options.ObjectExpression || options.ObjectPattern)) {
+    if (options &&
```

Comment:
Wondering if this could be replaced with something like `lodash.isPlainObject(options)`?

Reply:
Considering that the options can take different forms, more checks must be made for more node specific options.

I've made changes in https://github.com/eslint/eslint/pull/9876/commits/75bea253addf992294e48ba66b2fed55eccde402 to make the code more readable.

### Explanation:
```js
"never"
"always"

{ "multiline": true }
{ "minProperties": 2 }
{ "consistent": true }

// want to identify this
{ "ObjectExpression": "always" }
{ "ObjectExpression": { "multiline": true } }
```

1. find options that are objects. 
2. find the options that have a string or object as its value.